vm/rust/Makefile

```diff
@@ -1,6 +1,11 @@
 all:
 	cargo build --release
 
+debug:
+	cargo build && \
+	mkdir -p target/release/ && \
+	mv -n target/debug/libjuno_starknet_rs.* target/release/
```

Comment:
this might be confusing to both user and the CI caches. Do you think we can move the switch to our linker flags instead? ie `vm.go`

Reply:
Thanks, at first I was sceptical that it's possible to control cgo using `#ifdef` C preprocessor directives, but turned out it supports build constrints in form of `#cgo some_build_constraint_condition ...`